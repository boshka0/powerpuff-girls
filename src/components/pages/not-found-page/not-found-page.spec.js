import React from 'react';
import { HashRouter as Router } from 'react-router-dom';
import { mount } from 'enzyme';
import { renderToJson } from 'enzyme-to-json';

import NotFoundPage from './index';

it("render correctly error component", () => {
  const wrapper = mount(
    <Router>
      <NotFoundPage />
    </Router>
  );
  expect(renderToJson(wrapper.render())).toMatchSnapshot();
});
