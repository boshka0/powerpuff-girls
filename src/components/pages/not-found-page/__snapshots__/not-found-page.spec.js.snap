// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly error component 1`] = `
<div
  class="not-found"
>
  <a
    class="back-link"
    href="#/"
  >
    BACK
  </a>
  <p
    class="not-found__message"
  >
    Ooops nothing goes here...
  </p>
</div>
`;
